spec/rails_helper.rb

```diff
@@ -24,12 +24,12 @@
 
 Capybara.register_driver :headless_chrome do |app|
   capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
-    chromeOptions: { args: %w(headless disable-gpu) }
+    chromeOptions: { args: %w[headless disable-gpu] }
   )
 
   Capybara::Selenium::Driver.new app,
-    browser: :chrome,
-    desired_capabilities: capabilities
+                                 browser: :chrome,
+                                 desired_capabilities: capabilities
```

Comment:
We do not want to align params, because it adds noise in PRs when one variable changes and multiple lines need to be re-aligned. 

I think the rule for that is:
```
Layout/AlignParameters:
  Enabled: false
```

Reply:
Added that rule to my rubocop config and fixed the alignment issue 👍